tests/lib/snaps/test-snapd-location-control-provider/consumer

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+
+import dbus
+import os
+import sys
+
+INTERFACE = 'com.ubuntu.location.Service'
```

Comment:
IMHO This test is not realistic: if we want to pretend we are using the properties interface (get/set) then we should use the right interface here:
```
INTERFACE ='org.freedesktop.DBus.Properties'
```